files/default/sentry.rb

```diff
@@ -28,8 +28,15 @@ def report
         return if success?
         Raven.logger.info "Logging run failure to Sentry server"
         if exception
-          exception = sanitize_exception(exception)
-          evt = Raven::Event.capture_exception(sanitize_exception(exception))
+          # Why can't I make this a method? It makes 'exception' nil upon passing it in
+          # def sanitize_exception(exception)
+          #   exception
+          # end
+          if SANITIZE_WORD_LIST.any? { |word| exception.to_s.include?(word) }
```

Comment:
Unless the exception passed:
* Isn't of class Exception
* Overrides the to_s method with something that doesn't actually returning a string